plus/src/main/java/org/apache/calcite/chinook/CalciteConnectionProvider.java

```diff
@@ -52,7 +52,7 @@ private String provideSchema() throws IOException {
     final InputStream stream =
         getClass().getResourceAsStream("/chinook/chinook.json");
     requireNonNull(stream, "stream");
-    return CharStreams.toString(new InputStreamReader(stream, Charsets.UTF_8));
+    return CharStreams.toString(new InputStreamReader(stream, StandardCharsets.UTF_8));
```

Comment:
CalciteConnectionProvider.java:55: warning: [deprecation] UTF_8 in com.google.common.base.Charsets has been deprecated return CharStreams.toString(new InputStreamReader(stream, Charsets.UTF_8)); ^ error: warnings found and -Werror specified error: warnings found and -Werror specified